index.html

```diff
@@ -1,14 +1,213 @@
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<head>
+  <title>Mockup Home</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href="style.css" rel='stylesheet' type="text/css">
+  <link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@300&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="container">
+    <div class="item header">
+      <div class="header-wrapper">
+        <h1>HOME PAGE</h1>
+        <h4 id="h4-section">THE PAGE FOR THE HOME.</h4>
+      </div>
+      <div class=separation-content></div>
+    </div>
+    <div class="content-1">
+      <h3 class="div-h3"><span>TAKE A LOOK AT THIS AMAZING</span><span id="dark"> HOME PAGE!</span></h3>
+      <div class="images-container-content1">
+        <div class="image-wrapper-content-1">
+          <img class="image" src="https://placekitten.com/408/287" title="Another Kitten" alt="Another Kitten">
+          <p class="image-caption">KITTEN</p>
+        </div>
+        <div class="image-wrapper-content-1">
+          <img class="image" src="https://placekitten.com/408/287" title="Another Kitten" alt="Another Kitten">
+          <p class="image-caption">KITTEN</p>
+        </div>
+        <div class="image-wrapper-content-1">
+          <img class="image" src="https://placekitten.com/408/287" title="Another Kitten" alt="Another Kitten">
+          <p class="image-caption">KITTEN</p>
+        </div>
+      </div>
+      <div class="button">
+        <a class="cta" href="./continue-reading.html"><img class="image-cta" src="./button-img.png" title="CTA"> TELL ME
+          MORE</a></div>
+    </div>
+    <div class="content-2">
+      <h2>WHO DOESN'T LOVE KITTENS?</h2>
+    </div>
+    <div class="content-3">
+      <div>
+        <h3><span>I DON'T WANT TO SAY</span><span id="dark"> IT'S THE ALIENS ...</span></h3>
+      </div>
+      <div>
+        <h3>BUT IT'S THE ALIENS</h3>
+      </div>
+      <div><img class="image-post" src="https://placekitten.com/408/287" title="Another Kitten" alt="Another Kitten">
+      </div>
+      <div>
+        <h4 id="h4-content">Totally...</h4>
+      </div>
+      <div>
+        <p class="parag">Lorem ipsum dolor sit amet consectetur adipisicing elit. A perferendis, temporibus error
+          voluptatem, nemo ad
+          non provident nisi quod fuga nam tempora ducimus ex, pariatur at impedit quos voluptatum officia! Lorem ipsum
+        </p>
+      </div>
+      <div><a class="cta" href="./continue-reading.html"><img class="image-cta" src="./button-img.png" title="CTA"> CONTINUE
+          READING</a></div>
+    </div>

```

Comment:
`img` and the text underneath it aren't ending in the same spot.  `<p class="parag">` on line 58 spans the entire way across the container, but your picture does not;  consider rescaling your image to the full width.  

Also, having a `<p>` tag inside  a `<div>` like it is here is not really necessary.  It doesn't help with formatting, and overcomplicates your HTML!

Reply:
In fact, most of these don't need a separate `<div>` to properly use a flex container.  You can simply have your `flex items` grouped together in a single wrapper `<div>` in order to properly format them.